principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
```

Comment:
Roman numbers to aid translation and refer?

Reply:
Why Roman numbers? 
Does not hurt, not clear how it would help.

Reply:
Eric brought it up yesterday as a way to ease future translations. I.e. you can be sure you refer to the same principle in multiple languages by using it. IIRC this is a convention used in some international statues.